styles.css

```diff
@@ -9,721 +11,847 @@ body {
+  }
+
+  .myself {
+    margin-top: 6.25rem;
+    height: 990px;
+    margin-bottom: 80px;
+    padding: 20px 24px 0 24px;
+    background-color: #fff;
+    border-radius: 0 70px 0 0;
+  }
+
+  .about {
+    margin-bottom: 0.938rem;
+    margin-top: 3.125rem;
+    margin-left: 0;
+    font-size: 2.5rem;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 900;
+    color: #172b4d;
+  }
+
+  .experience {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.188rem;
+    font-weight: thin;
+    margin-bottom: 0.625rem;
+    color: #344563;
+  }
+
+  .connect-1 {
+    height: 3.125rem;
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 26px;
+  }
+
+  .icons-1 {
+    height: 3.125rem;
+    display: flex;
+    justify-content: space-evenly;
+    list-style-type: none;
+    margin-right: 5rem;
+  }
+
+  .icons-2 {
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .resume {
+    margin-bottom: 1.125rem;
+    border-radius: 0.5rem;
+    font-size: 1.188rem;
+    width: 9.375rem;
+    height: 3.125rem;
+    border: 0.063rem solid blue;
+    background-color: #fff;
+    color: #396df2;
+  }
+
+  .resume:hover {
+    background-color: #afc1f0;
+  }
+
+  .languages-1 {
+    font-size: 23px;
+    color: #000;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    margin-bottom: 25px;
+  }
+
+  #languages-23 {
+    background-image: url('images/Union-down.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  }
+
+  .language-icons {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    gap: 12px;
+  }
+
+  .language-icons li {
+    display: flex;
+    align-items: center;
+    background-color: #f7f7f9;
+    padding: 9px 12px;
+    border-radius: 12px;
+  }
+
+  .language-icons li p {
+    margin-left: 16px;
+  }
+
+  .alternative {
+    width: 70vw;
+  }
+
+  .language-container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .languages-below {
+    list-style-type: none;
+  }
+
+  .languages-below li {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 88%;
+  }
+
+  .frame {
+    font-family: 'Poppins', sans-serif;
+    margin-top: 35px;
+    font-size: 1.563rem;
+    font-weight: 500;
+    height: 2.5rem;
+    border-bottom: 0.063rem solid rgb(186, 183, 183);
+  }
+
+  /* #frame-1 {
+    background-image: url('images/Union.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  }
+
+  #skill-1 {
+    background-image: url('images/Union.svg');
+    background-repeat: no-repeat;
+    background-position: 18rem;
+  } */
+
+  .skills {
+    font-family: 'Poppins', sans-serif;
+    margin-top: 15px;
+    font-size: 1.563rem;
+    font-weight: 500;
+    height: 3.125rem;
+    margin-bottom: 0.938rem;
+    border-bottom: 0.063rem solid rgb(186, 183, 183);
+  }
+
+  .contact-form {
```

Comment:
- [ ] Kindly center the form well
- Your Design
<img width="1217" alt="Screenshot 2023-05-12 at 11 21 08" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/c006c41f-89dd-426d-8470-4f3e385df781">

- Figma Design

<img width="1440" alt="Screenshot 2023-05-12 at 11 23 40" src="https://github.com/MarkYegon7/Portfolio-Setup/assets/81444338/c905296e-d5b2-4a43-b415-40ace0032994">

Reply:
![Screenshot from 2023-05-13 00-09-14](https://github.com/MarkYegon7/Portfolio-Setup/assets/85193476/9cf82634-5c77-43ed-a949-a9ca32ec5044)
The contact form is centered.